### PR TITLE
ghostwriter: version bump

### PR DESCRIPTION
**Pull request by kenmays**

### CODE REVIEW

File: app-text/ghostwriter/ghostwriter-1.7.3.recipe [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+BUILD()
+{
+	lrelease ghostwriter.pro
+	qmake
+	make $jobArgs
+}
+
+INSTALL()
+{
+	mkdir -p $appsDir
```
Comment (extrowerk) [inline app-text/ghostwriter/ghostwriter-1.7.3.recipe]: You can do it in one line like
```mkdir -p $appsDir $dataDir/ghostwriter/translations```

File: app-text/ghostwriter/ghostwriter-1.7.3.recipe [hunk 2]
```diff
@@ -0,0 +1,91 @@
+SUMMARY="A cross-platform, aesthetic, distraction-free Markdown editor"
+DESCRIPTION="ghostwriter is a text editor for Markdown."
+HOMEPAGE="https://wereturtle.github.io/ghostwriter/"
+COPYRIGHT="wereturtle et al."
+LICENSE="GNU GPL v3"
+REVISION="1"
+SOURCE_URI="https://github.com/wereturtle/ghostwriter/archive/v$portVersion.tar.gz"
+CHECKSUM_SHA256="34eb3855c5cf7c541128eb740ee41de4000c91f71e9a2cf464c35d77dd523c79"
+ADDITIONAL_FILES="ghostwriter.rdef.in"
```
Comment (extrowerk) [inline app-text/ghostwriter/ghostwriter-1.7.3.recipe]: You forgot to add the rdef.in file.

Comment (kenmays) [reply app-text/ghostwriter/ghostwriter-1.7.3.recipe]: Its in https://github.com/kenmays/haikuports/blob/ghostwriter-1.7.3/app-text/ghostwriter/additional-files/ghostwriter.rdef.in

Comment (extrowerk) [reply app-text/ghostwriter/ghostwriter-1.7.3.recipe]: It needs to be in this PR.

Comment (extrowerk) [reply app-text/ghostwriter/ghostwriter-1.7.3.recipe]: Ok, i see, it is already in the tree: https://github.com/haikuports/haikuports/tree/master/app-text/ghostwriter/additional-files

Comment (extrowerk) [reply app-text/ghostwriter/ghostwriter-1.7.3.recipe]: Then remove the old recipe.

**Comment by extrowerk** [comment]
Looks ok, but not tested.

File: app-text/ghostwriter/ghostwriter-1.7.3.recipe [hunk 3]
```diff
@@ -5,7 +5,7 @@ COPYRIGHT="wereturtle et al."
 LICENSE="GNU GPL v3"
 REVISION="1"
 SOURCE_URI="https://github.com/wereturtle/ghostwriter/archive/v$portVersion.tar.gz"
-CHECKSUM_SHA256="bfd64024d9741f55d587afa5661f4b5e2a48becbbc92495c2186fca44fb08486"
+CHECKSUM_SHA256="34eb3855c5cf7c541128eb740ee41de4000c91f71e9a2cf464c35d77dd523c79"
 ADDITIONAL_FILES="ghostwriter.rdef.in"
```
Comment (alaviss) [inline app-text/ghostwriter/ghostwriter-1.7.3.recipe]:
```diff
 CHECKSUM_SHA256="34eb3855c5cf7c541128eb740ee41de4000c91f71e9a2cf464c35d77dd523c79"
+SOURCE_FILENAME="ghostwriter-$portVersion.tar.gz"
 ADDITIONAL_FILES="ghostwriter.rdef.in"
```

This change is to make the recipe conform to our [guidelines][0]

[0]: https://github.com/haikuports/haikuports/wiki/HaikuPorter-Guidelines#archive-name

Comment (kenmays) [reply app-text/ghostwriter/ghostwriter-1.7.3.recipe]: Done

**Pull request merged**